app/controllers/catalog_controller.rb

```diff
@@ -178,41 +191,57 @@ class CatalogController < ApplicationController
     # solr request handler? The one set in config[:default_solr_parameters][:qt],
     # since we aren't specifying it otherwise.
 
-    config.add_search_field 'all_fields', label: 'All Fields'
+    config.add_search_field 'all_fields', label: 'Keyword'
 
     # Now we see how to over-ride Solr request handler defaults, in this
     # case for a BL "search field", which is really a dismax aggregate
     # of Solr search fields.
 
     config.add_search_field('title') do |field|
-      # solr_parameters hash are sent to Solr as ordinary url query params.
       field.solr_parameters = {
-        'spellcheck.dictionary': 'title',
         qf: '${title_qf}',
         pf: '${title_pf}'
       }
+      field.solr_adv_parameters = {
+        qf:  '$qf_title',
+        pf:  '$pf_title'
+      }
     end
 
     config.add_search_field('author') do |field|
+      field.label = 'Author/Creator'
       field.solr_parameters = {
         'spellcheck.dictionary': 'author',
-        qf: '${author_qf}',
-        pf: '${author_pf}'
+        qf: "'${author_qf}'",
+        pf: "'${author_pf}'"
       }
     end
 
     # Specifying a :qt only to show it's possible, and so our internal automated
     # tests can test it. In this case it's the same as
-    # config[:default_solr_parameters][:qt], so isn't actually neccesary.
+    # config[:default_solr_parameters][:qt], so isn't actually necessary.
     config.add_search_field('subject') do |field|
       field.qt = 'search'
       field.solr_parameters = {
         'spellcheck.dictionary': 'subject',
-        qf: '${subject_qf}',
-        pf: '${subject_pf}'
+        qf: "'${subject_qf}'",
+        pf: "'${subject_pf}'"
+      }
+    end
+
+    config.add_search_field('identifiers') do |field|
+      field.include_in_simple_select = false
+      field.label = 'ISBN/ISSN'
+      field.solr_parameters = {
+        qf:  '$qf_number'
       }
     end
 
+    ## Stubs for Advanced Search as per https://github.com/psu-libraries/psulib_blacklight/wiki/Advanced-Search
+    # config.add_search_field('Series')
+    # config.add_search_field('Publisher')
+    # config.add_search_field('Publication date')
```

Comment:
`Publish Date` search from advanced is failing because `config.add_search_field('Publication date')` needs to be implemented, right? I think we talked about it but I just wanted to confirm. This is to be handled with the publication date changes.

Reply:
Didn't test that one, again, mainly because I am not sure what fields will end up in our list. Here's the error I see when testing that facet:

```
2019-02-01 13:44:35.925 ERROR (qtp1107530534-16) [   x:blacklight-core] o.a.s.h.RequestHandlerBase org.apache.solr.common.SolrException: undefined field pub_date_facet_field
```

Reply:
We changed this to `pub_date_ssim`

Reply:
My vote is to not fix this unless it turns out that we need to support it based on Ruth's input. What do you think?

Reply:
Yes our field is `pub_date_ssim`. 
I just checked again and publication date is listed on the advanced search fields on wiki. 
Saying that I think it does not have to be fixed with this PR, sth I need to fix with publication fields changes, right? I just wanted to make sure if you are also getting an error and also wanted to confirm that this is to be fixed later.

Reply:
There are just a lot ways we could handle publication date search/facet. Exact date with checkbox? Checkbox range? A true range picker like many other institutions have adopted? Etc.